src/main/java/nextstep/subway/domain/Sections.java

```diff
@@ -108,7 +143,7 @@ private void validateContainsNotAllStation(Station upStation, Station downStatio
 
     private void validateContainsAnyStation(Station upStation, Station downStation) {
         List<Station> stations = getStations();
-        if (stations.size() == 0) return;
+        if (stations.size() == ZERO) return;
```

Comment:
다음과 같이 isEmpty() 를 사용할 수도 있을 것 같아요 😄 
```suggestion
        if (stations.isEmpty()) {
            return;
        }
```